wcfsetup/install/files/lib/system/form/builder/container/wysiwyg/WysiwygFormContainer.class.php

```diff
@@ -316,6 +338,17 @@ public function isRequired() {
 		return $this->required;
 	}
 	
+	/**
+	 * Returns `true` if the preview-button should be generated automatically and returns `false` otherwise.
+	 * By default, the preview-button is generated automatically.
```

Comment:
```suggestion
	 * Returns `true` if the preview button will be shown and returns `false` otherwise.
	 * 
	 * By default, the preview button is shown.
```